en/docs/guides/login/integrate-custom-protocol.md

```diff
@@ -4,32 +4,31 @@ WSO2 Identity Server provides a way to decouple authentication functionality fro
 
 With this approach, the logic for processing an inbound authentication request will be written as an OSGi component (pluggable Java artifacts) called an inbound authenticator. To integrate an application which is not written based on a standard protocol, you can write a custom inbound authenticator and plug it to WSO2 Identity Server.
 
----
-
-This guide assumes you have your own web application and a custom inbound authenticator. If you wish to try out this flow with a sample application, click the button below. 
-
-<a class="samplebtn_a" href="../../../quick-starts/webapp-custom-sample" rel="nofollow noopener">Try it with the sample</a>
+<!---
 
-----
+---
 
-{!fragments/deploying-sample-apps.md!}
+This guide assumes you have your own web application and a custom inbound authenticator. If you wish to try out this flow with a sample application, click the button below.--->
+<!--- a class="samplebtn_a" href="../../../quick-starts/webapp-custom-sample" rel="nofollow noopener">Try it with the sample</a ---> 
```

Comment:
Has this file been added back to the nav?

Reply:
as discussed offline, this file (quick-starts/webapp-custom-sample) is hidden from mkdocs.yml file